addons/block_code/ui/blocks/utilities/drag_drop_area/drag_drop_area.gd

```diff
@@ -1,5 +1,4 @@
 @tool
-class_name DragDropArea
```

Comment:
What's the effect of this change – does it prevent the class being visible to the end user?

Reply:
With this, the class is not registered by name in ClassDB and it can't be referenced directly in other scripts, it must be preloaded. Also, if the script extends Node or a subclass of Node, it won't appear in the UI to the end user when adding a node.

Reply:
Another advantage: it won't collide with classes defined by the end user in the project. Our current Types class has a good chance to conflict given the generic name.

Reply:
I'll add this explanation to the commit message.